src/helpers/helpers.volumerendering.js

```diff
@@ -70,14 +73,16 @@ export default class HelpersVolumeRendering extends HelpersMaterialMixin(THREE.O
     this._uniforms.uPixelType.value = this._stack.pixelType;
     this._uniforms.uBitsAllocated.value = this._stack.bitsAllocated;
     this._uniforms.uPackedPerPixel.value = this._stack.packedPerPixel;
-    this._uniforms.uWindowCenterWidth.value = [offset + this._stack.windowCenter, this._stack.windowWidth * 0.8];
+    this._uniforms.uWindowCenterWidth.value = [this._windowCenter, this._windowWidth];
```

Comment:
I believe we should apply the offset here

Reply:
The reason I'd rather just apply it when passing to the shaders is that from a user perspective, they can pass negative values and do not have to think about the offset.